envoy/network/connection.h

```diff
@@ -194,8 +194,8 @@ class Connection : public Event::DeferredDeletable,
   /**
    * @return the address provider backing this connection.
```

Comment:
Can you update the doc comments now on the interfaces just so we don't forget later?

Reply:
yes